src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -255,6 +255,7 @@ def start_transfer(transfer_name, transfer_type, accession, paths, row_ids):
                 type=transfer_type, accession=accession,
                 transfer_metadata_set_row_uuid=row_id)
         except Exception:
+            logging.exception("Error copying %s to start of transfer", filepath)
```

Comment:
I think we'd want this to usually be printed; at `info` there's a good chance that many installations would be configured to not show it.